### ISSUE TITLE
Repeated local registry my-busybox test failures

### ISSUE THREAD

**Issue by dtrudg** [apptainer/singularity#5822]
All recent PRs are failing in e2e with:

```
=== CONT  TestE2E/PAR/DOCKER/registry
    singularitycmd.go:683: Running command "/usr/local/bin/singularity build --nohttps /tmp/stest.277559311/registry-589826795/container /tmp/TestTemplate-389160782"
    singularitycmd.go:683: 
        "/usr/local/bin/singularity build --nohttps /tmp/stest.277559311/registry-589826795/container /tmp/TestTemplate-389160782" output:
        INFO:    Starting build...
        FATAL:   While performing build: conveyor failed to get: Error reading manifest latest in localhost:5000/my-busybox: manifest unknown: manifest unknown
        
    singularitycmd.go:683: got 255 as exit code and was expecting 0: exit status 255
        waiting for command "/usr/local/bin/singularity build --nohttps /tmp/stest.277559311/registry-589826795/container /tmp/TestTemplate-389160782"
        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
        	github.com/sylabs/singularity/e2e/internal/e2e/singularitycmd.go:658
        github.com/sylabs/singularity/e2e/internal/e2e.Privileged.func1
        	github.com/sylabs/singularity/e2e/internal/e2e/priv_linux.go:45
        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func2
        	github.com/sylabs/singularity/e2e/internal/e2e/singularitycmd.go:683
        runtime.goexit
        	runtime/asm_amd64.s:1373
    singularitycmd.go:683: Running command "/usr/local/bin/singularity build --nohttps /tmp/stest.277559311/registry-589826795/container /tmp/TestTemplate-714829647"
    singularitycmd.go:683: Running command "/usr/local/bin/singularity build --nohttps /tmp/stest.277559311/registry-589826795/container /tmp/TestTemplate-947700196"
```

**Comment by dtrudg** [comment]
Registry instance is showing a fatal error from skopeo. The version installed now needs a trust policy that is not being installed.

```
time="2021-02-10T14:57:13Z" level=fatal msg="Error loading trust policy: open /etc/containers/policy.json: no such file or directory"
```

**Comment by dtrudg** [comment]
Closed in #5823
